shard-operator-parent/shard-operator-v2/src/main/java/com/redhat/service/smartevents/shard/operator/v2/ManagedProcessorServiceImpl.java

```diff
@@ -16,4 +25,9 @@ public void createManagedProcessor(ProcessorDTO processorDTO) {
     public void deleteManagedProcessor(ProcessorDTO processorDTO) {
         // @TODO
     }
+
+    @Override
+    public List<ManagedProcessor> fetchAllManagedProcessors() {
```

Comment:
@vaibhavjainwiz Could you add a unit test for this please. Thanks.

Reply:
I had the testcase for this class.